bedrock/firefox/views.py

```diff
@@ -443,7 +443,10 @@ def hello(request):
         'zh-TW': 'https://videos.cdn.mozilla.net/uploads/FirefoxHello/firefoxhello_intro_chinese'
     }
 
-    return l10n_utils.render(request, 'firefox/hello/index.html',
+    if (waffle.switch_is_active('firefox-hello-2016')):
```

Comment:
Ping @jgmize or @pmclanahan - new waffle switch here `firefox-hello-2016`. We don't need to enable it until March 8, but it would be great if we could have it set up & ready before then (else we could likely forget to do it).